extensions/control-plane/store/sql/contract-negotiation-store-sql/src/main/java/org/eclipse/edc/connector/store/sql/contractnegotiation/SqlContractNegotiationStoreExtension.java

```diff
@@ -33,9 +34,9 @@
 @Extension(value = "SQL contract negotiation store")
 public class SqlContractNegotiationStoreExtension implements ServiceExtension {
 
+    @Setting(value = "Name of the data source for contract negotiation")
```

Comment:
pls improve

Reply:
please add the `defaultValue` property as well